lib/ansible/modules/identity/ipa/ipa_user.py

```diff
@@ -49,12 +49,21 @@
     - List of telephone numbers assigned to the user.
     - If an empty list is passed all assigned telephone numbers will be deleted.
     - If None is passed telephone numbers will not be checked or changed.
+    version_added: 2.5
```

Comment:
Telephonenumber is already added in version 2.3.

Reply:
Uups. Sorry, should be fixed now.